test/integration/readme.integration.test.js

```diff
@@ -40,7 +38,24 @@ describe('Readme integration tests', () => {
 
   // jest --runInBand should be set to run the test cases in order
   beforeAll(async () => {
-    await axios.post(`${process.env.WIREMOCK_URL}/__admin/scenarios/reset`); // Reset Wiremock states
+    const wiremockURL = new URL(process.env.WIREMOCK_URL);
+    const request = http.request({
+      host: wiremockURL.hostname,
+      port: wiremockURL.port,
+      path: '/__admin/scenarios/reset',
+      method: 'POST',
+    });
+    request.end();
+    const p = new Promise((resolve, reject) => {
+      request
+        .on('response', (resp) => {
+          resolve(resp);
+        })
+        .on('error', (err) => {
+          reject(err);
+        });
+    });
+    await p; // Reset Wiremock states
```

Comment:
Do you think we need any `.then()` or `catch()` here?

Actually it looks like [Jest can handle the promise](https://jestjs.io/docs/setup-teardown#one-time-setup) so maybe it makes sense to remove the `await` (and the corresponding `async` on the function) and just return the promise (either calling `setAuthentication()` beforehand or chaning it in a `then()`).

Reply:
done in [50055c9](https://github.com/IBM/cloudant-node-sdk/pull/710/commits/50055c9ab06d9dd00ddcac5ac77ce6af77bb7e7e)